content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -260,7 +260,8 @@ The scenario is a GitHub organization owner can authorize its organization membe
         - id: kubeflow-oidc-authservice
           redirectURIs: ["/login/oidc"]
           name: 'Dex Login Application'
-          secret: pUCnCOY80SnXgjibTYM0ZWNzY3xreNGQok
+          # Update the secret below to match with the oidc authservice.
+          secret: pUBnBOY80SnXgjibTYM9ZWNzY2xreNGQok
```

Comment:
👍 it matches that from https://github.com/kubeflow/manifests/blob/14de0c7dc27231c0a5840f99f125acc73a351ce1/stacks/ibm/application/oidc-authservice/kustomization.yaml#L20